ansible/roles/atmo-mount-volume/tasks/main.yml

```diff
@@ -16,9 +15,17 @@
     fstype: '{{ VOLUME_DEVICE_TYPE }}'
     src: '{{ VOLUME_DEVICE }}'
     state: 'mounted'
-    name: '{{ VOLUME_MOUNT_LOCATION }}'
+    path: '{{ VOLUME_MOUNT_LOCATION }}'
   register: mount_result
 
 - name: Debug mount result
   debug:
     var: mount_result
+
+- name: Change directory ownership
+  file:
+    state: directory
+    path: "{{ VOLUME_MOUNT_LOCATION }}"
+    recurse: yes
+    owner: "{{ ATMOUSERNAME }}"
+    group: "iplant-everyone"
```

Comment:
Lets not hard-code the group here, we will want a different value for other deployers of `atmosphere-ansible`